content/en/blog/_posts/2025-05-07-kubernetes-1-33-wi-for-image-pulls.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: blog
-title: "From Secrets to Service Accounts: Kubernetes Image Pulls Evolved"
-date: 2025-04-23
+title: "Kubernetes v1.33: From Secrets to Service Accounts: Kubernetes Image Pulls Evolved"
+date: 2025-05-07
```

Comment:
While this is only a "date" field, we can provide the timestamp as well. That allows us to schedule things more consistently throughout the cycle, so I'd like the following changes made
```suggestion
date: 2025-05-07T10:30:00-08:00
```